src/libs/OptionsListUtils.ts

```diff
@@ -550,9 +550,15 @@ function getAlternateText(option: ReportUtils.OptionData, {showChatPreviewLine =
     if (!!option.isThread || !!option.isMoneyRequestReport) {
         return option.lastMessageText ? option.lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
     }
-    if (!!option.isChatRoom || !!option.isPolicyExpenseChat) {
+
+    if (option.isChatRoom) {
```

Comment:
I think we only need to do this for chat rooms that aren't `#admins` and `#announce`, for those I believe we still want to display the policy name